x-pack/legacy/plugins/index_management/public/app/sections/home/index_list/index_actions_context_menu/index_actions_context_menu.js

```diff
@@ -368,7 +368,7 @@ export class IndexActionsContextMenu extends Component {
           <EuiSpacer size="m" />
 
           <EuiForm
-            isInvalid={this.forcemergeSegmentsError()}
+            isInvalid={!!this.forcemergeSegmentsError()}
```

Comment:
Is it still possible for this error condition to come about given the `EuiFieldNumber` change?

Reply:
You could type a negative number so I think the validation is still necessary.